third_party/nvfuser/benchmark/matmul.cpp

```diff
@@ -101,16 +101,21 @@ std::pair<at::Tensor, at::Tensor> fp16MatmulAtInput(
     MatmulLayout layout) {
   auto options = at::TensorOptions().dtype(at::kHalf).device(at::kCUDA, 0);
 
+  // Use randint to reduce numerical error so we can easily validate the
+  // correctness of results.
   switch (layout) {
     case MatmulLayout::TT:
       return std::make_pair(
-          at::randn({M, K}, options), at::randn({K, N}, options));
+          at::randint(-3, 3, {M, K}, options).div_(8),
```

Comment:
Does this mean a 2D tensor of size `MxK` of values randomly generated between -3 and 3 and is then divided by 8? Why division by 8?

Reply:
Yes, you are right. I want values of 0.XX with small mantissa usage. Compared with multiplying `0.1`, 8 is a multiple of 2, so it uses less mantissa.

Reply:
Doesn't `at::randint` generate an integer tensor, and division by 8 would just result in zero?

Reply:
The dtype will be the dtype in `options`: See https://pytorch.org/docs/stable/generated/torch.randint.html